config.js

```diff
@@ -83,12 +83,20 @@ Config.prototype._seed = function _seed(seed) {
     seedOrDefault('dampScoringSuppressDuration', 60 * 60 * 1000); // 1 hr in ms
     seedOrDefault('dampScoringSuppressLimit', 5000);
 
+    seedOrDefault('lagSamplerEnabled', true);
+    seedOrDefault('lagSamplerInterval', 1000, numValidator);
+
     // Joiner config
     seedOrDefault('joinDelayMin', 100, numValidator); // ms
     seedOrDefault('joinDelayMax', 2 * 60 * 1000, numValidator); // 2 min in ms
     seedOrDefault('joinTroubleErrorEnabled', true);
     seedOrDefault('maxJoinDuration', 20 * 60 * 1000, numValidator); // 20 mins in ms
 
+    // Forwarding config
+    seedOrDefault('backpressureEnabled', true);
+    seedOrDefault('maxEventLoopLag', 1500, numValidator);
+    seedOrDefault('maxInflightRequests', 1000, numValidator);
```

Comment:
It's a fair point. I had the intention to be fairly conservative with these numbers so as not to disrupt current expectations by default. But, even 1k might be too low. Let me revisit.